### PR TITLE
Refactor typography styles to use helper functions

### PR DESCRIPTION
**Pull request by andre1050**
## What does this change?
Refactors all our components to use all of the typography styles from a series of 3 helpers we created in `typography.ts`, which mirror the typography helpers provided by @guardian/design-system. This lets us clean up lots of typography related CSS we had lying around and keep all of out typography stuff in one please.
Eventually, we'll delete all our helpers from this file and revert back to use directly the design system's. We can't do that yet because the font sizes and line heights are different and we haven't decided yet to adopt the design system convention.

## Why?
Because it was a pain to copy & paste font styles everywhere and because all the repeated typography CSS was cluttering out components.

## Link to supporting Trello card
https://trello.com/c/U6P8tHxo/69-refactor-components-to-use-fonts-from-the-design-system

### CODE REVIEW

File: src/components/buttons/ContinueButton.tsx [hunk 1]
```diff
@@ -10,10 +9,10 @@ interface Props {
 // Splitting the CSS for the button link into two separate objects;
 // First, all the standard LinkCSS properties we need, so they can be typechecked...
 const anchorStyles: LinkCSS = {
-    color: "#ffffff",
     fontFamily: "'Guardian Text Sans',sans-serif",
     fontSize: "17px",
     lineHeight: "36px",
+    color: "#ffffff",
```
Comment (nicl) [inline src/components/buttons/ContinueButton.tsx]: TODO for us (but we can do in a different PR) to use palette colours here, rather than inlined hex values.

Comment (andre1050) [reply src/components/buttons/ContinueButton.tsx]: Good spot, I've made the change.
